packages/components/src/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,287 @@
+import React from 'react';
+import { Col, FormControl, Row } from 'react-bootstrap';
+
+import { List } from 'immutable';
+
+import { SectionHeading } from '../SectionHeading';
+
+import { DomainFieldLabel } from '../DomainFieldLabel';
+import { Principal, SelectInput } from '../../..';
+
+import { IssuesListDefModel } from './models';
+import {UserGroup} from "../../permissions/models";
+import {
+    ISSUES_LIST_DEF_SORT_DIRECTION_TIP,
+    ISSUES_LIST_DEF_SINGULAR_PLURAL_TIP,
+    ISSUES_LIST_GROUP_ASSIGN_TIP, ISSUES_LIST_USER_ASSIGN_TIP
+} from "./constants";
+import { getCoreGroups, getCoreUsersInGroups } from "../../permissions/actions";
+
+interface IssuesListDefBasicPropertiesInputsProps {
+    model: IssuesListDefModel;
+    onInputChange?: (any) => void;
+    onSelect?: (selected: String, name?: String) => void;
+}
+
+interface SecurityUserGroupProps {
+    model: IssuesListDefModel;
+    coreGroups?: List<Principal>;
+    onSelect?: (selected: Principal, name?: String) => any;
+    coreUsers?: List<UserGroup>;
```

Comment:
I was a little confused when I saw that `coreGroups` and `coreUsers` were still in the Props here since you have moved them to the State interface below, but I see that it is because they are used as props in the actual input components in this same file. I might suggest either creating new Prop interfaces for those input components or adding a comment here to these props.

Reply:
I've cleaned up the interface names and such to make things clear. Plus comment now added. let me know what you think!